frontend/public/components/dashboard/capacity-card/capacity-card.scss

```diff
@@ -13,8 +13,10 @@
 }
 
 .co-capacity-card__item-description {
-  font-size: 0.6875em;
+  font-size: 0.875rem;
   margin-bottom: 0;
+  font-weight: normal;
```

Comment:
nit: we try to keep the css declarations alphabetized

Reply:
Ah good to know, fixed, thanks!